### DISCUSSION TITLE
停贷潮已经快一年了，各位的楼盘复工了没有？

### DISCUSSION THREAD

**Discussion by ytx0574** [WeNeedHome/SummaryOfLoanSuspension#1207] [General]
我的房子情况是主体建好， 电梯/绿化/车库还没搞。

我也是停贷潮久开始停的， 至今没还过钱。 保交楼的政策出了， 但我这个响应很滑稽。 去年的10月搞过复工仪式， 假的。  很多业主没还贷， 由于保交楼的政策， 目前我的盘已经被划给另外一个开发商来续建，目前在等车库图纸， 每天工地就三十个人抓毛，算是有一点进度吧。。。

**Comment by ytx0574** [comment]
> 我也是停贷潮久开始停的， 至今没还过钱。 保交楼的政策出了， 但我这个响应很滑稽。 去年的10月搞过复工仪式， 假的。 很多业主没还贷， 由于保交楼的政策， 目前我的盘已经被划给另外一个开发商来续建，目前在等车库图纸， 每天工地就三十个人抓毛，算是有一点进度吧。。。

但从烂尾开始久划给金科续建， 它是各种甩锅， 反正就是哪哪儿都难， 没有态度😭

**Comment by Leon0312** [comment]
和楼上相同， 三线城市， 地方政府说有资金进来， 银行打电话说征信暂时不受影响。 但是进度缓慢，

**Reply by ytx0574** [reply]
我在想要是房价不上涨, 开发商根本久没想法搞, 能拖一天是一天

**Comment by itsoursduty** [comment]
别指望了。

**Comment by dinglino1** [comment]
银行不会强制从你的账户扣款吗

**Reply by ytx0574** [reply]
不往那张卡存钱, 目前没扣.  不过信用卡/花呗给停了

**Comment by liyan0628** [comment]
来信已收到，谢谢！

**Comment by Cegazhos** [comment]
希望更多的懦夫能跟你一样 不妥协
